### ISSUE TITLE
Getting comments with pagination

### ISSUE THREAD

**Issue by Aerox1** [mgp25/Instagram-API#684]
Hello . I have been struggling with getting high number of comments for couple of days . When the count is about 1k everything is fine. But when the number goes up suddenly in the middle of Pagination Loop the nexMaxId becomes null .  I could do some improvements with setting sleep in the loop but its not still perfect . For example for 18k comments without sleep i could get only 2k comments . with sleep once i got 15k and another time 17k . But i takes too much time . 

I just wanted to know your idea about why this would happen and is there a better way for handling this rather than sleep or not .  

Best Regards

**Comment by Lastwisher** [comment]
Hello there, 

How many second do you use as sleep ? 

thanks.

**Comment by lucassimao** [comment]
I think it's not a API related problem, it's another way of how instagram backend tries to prevent scraping and to defeat bots. Some days ago I was sending DM to my store instagram's followers with the picture of my products. Every 1 hour, I share some posted picture with 5 followers through direct message. I iincreased the number of direct message to 10 and all of sudden some messages are not delivered, without error message or error status. So, I rolled back my code changes, and everything is working again ( sending only 5 direct message each hour). It's like playing a cat and mouse game
